iwf/command_results.py

```diff
@@ -40,6 +40,7 @@ class CommandResults:
     timer_commands: list[TimerCommandResult]
     internal_channel_commands: list[InternalChannelCommandResult]
     signal_channel_commands: list[SignalChannelCommandResult]
+    wait_until_api_succeeded: Optional[bool] = None
```

Comment:
I added this to match the properties in the `CommandResults` class in Java SDK: https://github.com/indeedeng/iwf-java-sdk/blob/main/src/main/java/io/iworkflow/core/command/CommandResults.java#L17-L23